src/xo-app/dashboard/overview/index.js

```diff
@@ -420,14 +429,17 @@ export default class Overview extends Component {
                   <Icon icon='vm-force-shutdown' /> {_('vmStatePanel')}
                 </CardHeader>
                 <CardBlock>
-                  <ChartistGraph
-                    data={{
-                      labels: ['Running', 'Halted', 'Other'],
-                      series: [this.props.vmMetrics.running, this.props.vmMetrics.halted, this.props.vmMetrics.other]
-                    }}
-                    options={{ showLabel: false }}
-                    type='Pie' />
-                  <p className='text-xs-center'>{this.props.vmMetrics.running} running ({this.props.vmMetrics.halted} halted)</p>
+                  <Link to='/home?s=power_state:running+&t=VM'>
```

Comment:
Currently, this chart does not support different clicks (running, not running...), it's a future feature. Now, I choose by default to redirect on running vms.

The VMs number link at the top of the page is used to get all vms. Not this link.